.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
this folder needs to be in .gitignore and this file removed

Reply:
Resolving this as the changes are approved by @naathanbrown